.prettierrc.json

```diff
@@ -1,20 +1,6 @@
 {
     "$schema": "http://json.schemastore.org/prettierrc",
-
-    "parser": "typescript",
-    "printWidth": 120,
-    "semi": true,
     "singleQuote": true,
-    "tabWidth": 4,
-    "trailingComma": "es5",
```

Comment:
Let's use `"all"`?

Reply:
"all" required a newer version of node to run by defautl - but given these are all parsed by TS, that seems fine 👍